src/content/reference/react/captureOwnerStack.md

```diff
@@ -0,0 +1,301 @@
+---
+title: captureOwnerStack
+---
+
+<Wip>
+
+**This API is experimental and is not available in a stable version of React yet.**
```

Comment:
I think we should update this to the Canary badge, and just wait until we add this to Canary to merge the docs.

Reply:
Makes sense. Need to remember to update the sandboxes to use the Canary channel. I'll leave them on experimental for now so that we can verify they're working.